ozaria/site/views/play/level/LevelGoals.vue

```diff
@@ -1,33 +1,31 @@
-<template lang="pug">
-  div
-    // TODO: Split this into two components, one the ul, the other the goals-status
-    ul#primary-goals-list(dir="auto")
-      level-goal(
-        v-for="goal in levelGoals",
-        :goal="goal",
-        :state="goalStates[goal.id]",
-      )
-    level-goal(
-      v-if="conceptGoals.length",
-      :goal="{ name: $t('play_level.use_at_least_one_concept') }",
-      :state="{ status: conceptStatus }",
-    )
-    ul#concept-goals-list(dir="auto" v-if="conceptGoals.length")
-      level-goal.concept-goal(
-        v-for="goal in conceptGoals",
-        :goal="goal",
-        :state="goalStates[goal.id]",
-      )
-      
-    div.goals-status.rtl-allowed(v-if="showStatus")
-      span {{ $t("play_level.goals") }}
-      span.spr :
-      span(v-if="classToShow === 'running'").goal-status.running {{ $t("play_level.running") }}
-      span(v-if="classToShow === 'success'").goal-status.success {{ $t("play_level.success") }}
-      span(v-if="classToShow === 'incomplete'").goal-status.incomplete {{ $t("play_level.incomplete") }}
-      span(v-if="classToShow === 'timed-out'").goal-status.timed-out {{ $t("play_level.timed_out") }}
-      span(v-if="classToShow === 'failing'").goal-status.failure {{ $t("play_level.failing") }}
-
+<template>
+  <div>
+    <div v-if="showStatus" class="goals-status rtl-allowed">
+      <span>{{ $t("play_level.goals") }} : {{ $t("play_level." + goalStatus) }}</span>
+    </div>
+    <div class="level-goals">
+      <!-- TODO: Split this into two components, one the ul, the other the goals-status-->
+      <ul id="primary-goals-list" dir="auto">
+        <level-goal v-for="goal in levelGoals"
+                    :key="goal.id"
```

Comment:
These keys don't prevent Vue warnings for some reason. Lmk if you know why this still isn't considered unique!

Reply:
I can't reproduce the warning on my environment. And my Vue debugger shows that keys are being correctly set.

Reply:
Interesting! It is in my build output from `npm run dev` that this is being warned about.